### ISSUE TITLE
[FR] add custom keybinds to effects

### ISSUE THREAD

**Issue by Caliel666** [wwmm/easyeffects#1477]
being able to set up keybinds to quick bypass effects would be nice, specially to use on macros and other software

**Comment by Caliel666** [comment]
Yes, more specifically i want to bypass effects like reverb in just a press of a button without having to open the program (it's running in the background) and without clicking anything

**Comment by wwmm** [comment]
>i want to bypass effects like reverb in just a press of a button without having to open the program

In that case the usual keybinding won't work because they need the window to be opened. What you want has to be done through scripts that access our database through the `GSettings` API. Take a look at https://github.com/wwmm/easyeffects/discussions/916. The instructions are for PulseEffects but the same idea is valid for EasyEffects.

**Comment by wwmm** [comment]
>because they need the window to be opened

And in the foreground.

**Comment by tomasz-brak** [comment]
> > i want to bypass effects like reverb in just a press of a button without having to open the program
> 
> In that case the usual keybinding won't work because they need the window to be opened. What you want has to be done through scripts that access our database through the `GSettings` API. Take a look at #916. The instructions are for PulseEffects but the same idea is valid for EasyEffects.

Is there an  option to enable a preset from the gsettings api?

**Comment by wwmm** [comment]
>Is there an option to enable a preset from the gsettings api?

No. But there is a command line option that allows a preset to be loaded. You can see the option executing `easyeffects -h`.
